HeavyIonsAnalysis/JetAnalysis/src/RecHitTreeProducer.cc

```diff
@@ -876,6 +890,7 @@ RecHitTreeProducer::beginJob()
     eeTree->Branch("perp",eeRecHit.perp,"perp[n]/F");
     eeTree->Branch("chi2",eeRecHit.chi2,"chi2[n]/F");
     eeTree->Branch("eError",eeRecHit.eError,"eError[n]/F");
+    eeTree->Branch("flags",eeRecHit.flags,"flags[n]/I");
```

Comment:
I think that "flags[n]/I" should instead be "flags[n]/i" (lowercase 'i' for unsigned).